libs/utils/src/lib/isTruthy.ts

```diff
@@ -0,0 +1,3 @@
+export type Truthy<T> = NonNullable<T>;
+
+export const isTruthy = <T>(value: T): value is Truthy<T> => Boolean(value);
```

Comment:
What do you think about creating an RxJS operator helper lib as well? 🤔 

```
 export function filterTruthy<T>(): (s$: Observable<T | null>) => Observable<T> {
  return s$ => s$.pipe(filter(isTruthy<T>))
}
```

Reply:
I don't see a big value in that
The difference between usages
```ts
myObs$.pipe(filter(isTruthy))
```

and

```ts
myObs$.pipe(filterTruthy)
```

Doesn't provide us with better readability or reduced boilerplate IMO